src/lights/PointLight.js

```diff
@@ -10,6 +10,7 @@ THREE.PointLight = function ( color, intensity, distance ) {
 
 	this.intensity = ( intensity !== undefined ) ? intensity : 1;
 	this.distance = ( distance !== undefined ) ? distance : 0;
+	this.decayExponent = ( decayExponent !== undefined ) ? decayExponent : 1;	// for physically correct lights, should be 2.
```

Comment:
It would be great to have physically-based light attenuation in ThreeJS as it is supported in high quality production renderers and other real-time renderers as well - such as Unreal Engine, Unity, Crytek.  The old model didn't have any form of exponential decay and in my opinion didn't look realistic.  I agreed to the new model because at least it had exponential control, which was better than the old model and it was fully backwards compatible, but I am compromising on it as I really did want a true physical falloff.

Unreal has a check box to enable Physical Falloff.  Might be the way to go.